main/ui/src/main/resources/localization.properties

```diff
@@ -13,6 +13,9 @@ main.directoryList.contextMenu.remove=Remove from list
 main.directoryList.contextMenu.changePassword=Change password
 main.addDirectory.contextMenu.new=Create new vault
 main.addDirectory.contextMenu.open=Open existing vault
+main.directoryList.remove.confirmation.title=Vault removal
+main.directoryList.remove.confirmation.header=Do you really want to remove this vault ?
+main.directoryList.remove.confirmation.content=Every data it contains will be lost.
```

Comment:
This file doesn't exist any longer in master, as we switched from .properties files to unicode .txt files due to charset issues with non-latin alphabets.
Please merge from master and put the new strings into `src/main/resources/localization/en.txt`